scripts/dev-integration-tests.sh

```diff
@@ -26,12 +26,24 @@ then
     echo "Please build the image before running integration tests"
     exit 0
 fi
+if [ "x$(docker images|grep networkservicemesh/nsm-simple-dataplane)" == "x" ]
+then
+    echo "Docker image networkservicemesh/nsm-simple-dataplane not found"
+    echo "Please build the image before running integration tests"
+    exit 0
```

Comment:
Do we want to exit 0 in these scenarios? May make sense to exit with a non zero result.